hipchat.go

```diff
@@ -8,7 +8,10 @@ import (
 	"net/http"
 )
 
-var notifyURL = "https://api.hipchat.com/v2/room/%s/notification?auth_token=%s"
+var (
+	defaultUrl = "https://api.hipchat.com"
```

Comment:
`Url` should be `URL` https://github.com/golang/go/wiki/CodeReviewComments#initialisms